cpp/CMakeLists.txt

```diff
@@ -82,6 +82,9 @@ endif(CMAKE_BUILD_TYPE MATCHES Debug)
 option(BUILD_TESTS "Configure CMake to build tests"
        ON)
 
+option(BUILD_BENCHMARKS "Configure CMake to build (google) benchmarks"
+       ON)
```

Comment:
I would argue we should default this to off.
```suggestion
       OFF)
```

Reply:
Yep, CI can choose to enable it when support is ready.